{"publishedDate": "2021-05-27T19:15Z", "lastModifiedDate": "2024-11-21T04:55Z", "cve": {"data_type": "CVE", "data_format": "MITRE", "data_version": "1.0", "CVE_data_meta": {"ID": "CVE-2020-10698", "ASSIGNER": "secalert@redhat.com"}, "description": {"description_data": [{"lang": "en", "value": "A flaw was found in Ansible Tower when running jobs. This flaw allows an attacker to access the stdout of the executed jobs which are run from other organizations. Some sensible data can be disclosed. However, critical data should not be disclosed, as it should be protected by the no_log flag when debugging is enabled. This flaw affects Ansible Tower versions before 3.6.4, Ansible Tower versions before 3.5.6 and Ansible Tower versions before 3.4.6."}, {"lang": "es", "value": "Se encontr\u00f3 un fallo en Ansible Tower cuando se ejecutan trabajos.&#xa0;Este fallo permite a un atacante acceder a la salida est\u00e1ndar de los trabajos realizados que son ejecutados desde otras organizaciones. Algunos datos confidenciales pueden ser divulgados.&#xa0;Sin embargo, los datos cr\u00edticos no deben divulgarse, ya que deben estar protegidos por el indicador no_log cuando la depuraci\u00f3n est\u00e1 habilitada. Este fallo afecta a Ansible Tower versiones anteriores a 3.6.4, Ansible Tower versiones anteriores a 3.5.6 y Ansible Tower versiones anteriores a 3.4.6"}]}, "references": {"reference_data": [{"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1818924", "name": "", "refsource": "", "tags": ["Issue Tracking", "Patch", "Vendor Advisory"]}, {"url": "https://bugzilla.redhat.com/show_bug.cgi?id=1818924", "name": "", "refsource": "", "tags": ["Issue Tracking", "Patch", "Vendor Advisory"]}]}, "problemtype": {"problemtype_data": [{"description": [{"lang": "en", "value": "CWE-200"}, {"lang": "en", "value": "NVD-CWE-Other"}]}]}}, "impact": {"baseMetricV2": {"cvssV2": {"version": "2.0", "vectorString": "AV:L/AC:L/Au:N/C:P/I:N/A:N", "baseScore": 2.1, "accessVector": "LOCAL", "accessComplexity": "LOW", "authentication": "NONE", "confidentialityImpact": "PARTIAL", "integrityImpact": "NONE", "availabilityImpact": "NONE"}, "exploitabilityScore": 3.9, "impactScore": 2.9, "severity": "LOW", "obtainAllPrivilege": false, "obtainUserPrivilege": false, "obtainOtherPrivilege": false, "userInteractionRequired": false}, "baseMetricV3": {"exploitabilityScore": 1.8, "impactScore": 1.4, "cvssV3": {"version": "3.1", "vectorString": "CVSS:3.1/AV:L/AC:L/PR:L/UI:N/S:U/C:L/I:N/A:N", "baseScore": 3.3, "baseSeverity": "LOW", "attackVector": "LOCAL", "attackComplexity": "LOW", "privilegesRequired": "LOW", "userInteraction": "NONE", "scope": "UNCHANGED", "confidentialityImpact": "LOW", "integrityImpact": "NONE", "availabilityImpact": "NONE"}}}, "configurations": {"CVE_data_version": "4.0", "nodes": [{"operator": "OR", "negate": false, "children": [], "cpe_match": [{"vulnerable": true, "cpe23Uri": "cpe:2.3:a:redhat:ansible_tower:*:*:*:*:*:*:*:*", "matchCriteriaId": "50566FBA-06F5-461A-A9A5-87E9FC98B085", "cpe_name": [], "versionEndExcluding": "3.4.6"}, {"vulnerable": true, "cpe23Uri": "cpe:2.3:a:redhat:ansible_tower:*:*:*:*:*:*:*:*", "matchCriteriaId": "90C46E3E-82AF-4E9F-AB9A-856988169454", "cpe_name": [], "versionStartIncluding": "3.5.0", "versionEndExcluding": "3.5.6"}, {"vulnerable": true, "cpe23Uri": "cpe:2.3:a:redhat:ansible_tower:*:*:*:*:*:*:*:*", "matchCriteriaId": "F1299B71-D05C-4871-ABC0-F70AA3730FFB", "cpe_name": [], "versionStartIncluding": "3.6.0", "versionEndExcluding": "3.6.4"}]}]}}